packages/html/src/action-sheet/actionsheet-header.tsx

```diff
@@ -7,8 +7,10 @@ export const ACTIONSHEETHEADER_CLASSNAME = `k-actionsheet-titlebar`;
 export type KendoActionSheetHeaderProps = {
     title?: string;
     subTitle?: string;
-    actions?: string[];
+    actionsStart?: string[];
```

Comment:
what if you have 1 action only? I don't this this would work. Also, I am not sure if actionsStart/end is the best name here- maybe actionsTitlebar and actionsFooter?